parser-combinators.cabal

```diff
@@ -1,5 +1,5 @@
 name:                 parser-combinators
-version:              0.1.0
+version:              0.1.1
```

Comment:
I'll revert this.

FWIW PVP says this when adding new modules:

> Adding new modules. Adding new modules might cause an unavoidable name collision in dependent code. However, this is usually pretty unlikely, especially if you keep to your own namespace, so only an increase of the minor version number is required, in other words A.B MAY remain the same the new C MUST be greater than the old C. If, however, your added module name is taken from another package (e.g. when network-bytestring was merged into network) or is quite general (Data.Set or something similar) then the version increase SHOULD be major.

So if I get this right it's OK to bump minor version.